osmUtils/utils_geo.py

```diff
@@ -0,0 +1,141 @@
+
+"""General utility functions."""
+from .settings import DEFAULT_CRS, DEFAULT_TILES
+import geopandas as gpd
+import pandas as pd
+import mercantile as mt
+from shapely.geometry import shape, MultiPolygon, Polygon
```

Comment:
A veryyy minor nitpick, but we should order imports with all the `from ... import ...` at the bottom to make it easier to read